centos-ci/jobs/gluster-kubernetes.yml

```diff
@@ -0,0 +1,44 @@
+- job:
+    name: gluster_gluster-kubernetes
+    node: gluster
+    description: Run the comples test from gluster-kubernetes. Tests are
+      triggered for a pull-request after one of the repository admins approves
+      it (with standard Jenkins GitHub commands).
+    project-type: freestyle
+    concurrent: true
+
+    scm:
+    - git:
+        url: https://github.com/gluster/glusterfs-patch-acceptance-tests.git
+        branches:
+        - origin/master
+
+    properties:
+    - github:
+        url: https://github.com/gluster/gluster-kubernetes/
+
+    triggers:
+    - github-pull-request:
```

Comment:
This entire section seems really error-prone. Is there no way to utilize GitHub's own API to limit acceptance of triggers to the current admins of the project?

Reply:
No, the plugin only allows to have everyone in an organization ('gluster' contains all teams that maintain Gluster Community projects).

Reply:
Oof.... ah well. We'll need to document this in the gk repo, then. :)